admin-dev/themes/new-theme/js/pages/currency/form/components/CurrencyFormatter.vue

```diff
@@ -46,6 +47,7 @@
 <script>
   import LanguageList from './LanguageList';
   import CurrencyModal from './CurrencyModal';
+  import ConfirmModal from '@components/modal';
```

Comment:
You are mixing a jquery modal within this vue component This should be avoided at all cost, there is a modal vue component that you should use instead:
`import Modal from '@vue/components/Modal';`

Reply:
Oh one thing that could be interesting is to create a `ConfirmModal` vue component (in the `vue/component` folder) It could be based on the already existing modal component and allows a few customization (like the jquery one):
- title
- button labels
- modal description content
- confirm/cancel callbacks

The good thing is that thanks to the slots and properties you can have default behaviour that are easily overridable. This is the kind of component we will definitely need in the future so I think it's an interesting topic :)